src/viewProviders/flinkArtifacts.ts

```diff
@@ -50,12 +39,52 @@ export class FlinkArtifactsViewProvider
             const loader = CCloudResourceLoader.getInstance();
             this._artifacts = await loader.getFlinkArtifacts(this.computePool!);
           } catch (error) {
-            this.logger.error("Failed to load Flink artifacts", { error });
+            let showNotification = false;
+            let message = "Failed to load Flink artifacts.";
+
+            if (isResponseError(error)) {
```

Comment:
followed the pattern[ here ](https://github.com/confluentinc/vscode/blob/main/src/flinkStatementResultsManager.ts#L289)for this